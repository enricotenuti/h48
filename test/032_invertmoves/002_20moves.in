B2 R D2 B2 R D2 F2 L2 F2 R' F2 D' L D2 L U2 L B2 F2 U
