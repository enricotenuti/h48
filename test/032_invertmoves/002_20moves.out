U' F2 B2 L' U2 L' D2 L' D F2 R F2 L2 F2 D2 R' B2 D2 R' B2
